scripts/train.py

```diff
@@ -51,7 +51,9 @@ def create_argparser():
         fp16_scale_growth=1e-3,
         few_shot_count=10,
         val_size=2,
-        target=None
+        target=None,
+        save_opt=True,
+        save_ema=True
     )
```

Comment:
do you want to be backward compatible without any flags? 
- then change the default values of `few_shot_count` and `val_size` to None or 0, depends on how you check for their existence.
you want to have default values for our own use?
- then change the default values of `save_opt` and `save_ema` to False.

Reply:
No. This file is configured for validation, but no one says that on default you do not want to save EMA and optimizer params even if you train for few-shot.
Besides, there are many other things that are not configured properly for out specific few-shot training (save interval, lr)

Reply:
not good :(